src/Model/ArrayData.php

```diff
@@ -102,6 +97,16 @@ public function hasField(string $fieldName): bool
         return isset($this->array[$fieldName]);
     }
 
+    public function exists(): bool
+    {
+        return !empty($this->array);
+    }
```

Comment:
Can't remember exactly why I needed this but it's a bit of a no-brainer to add - otherwise it'll always return `true` (per the superclass) even when there's no data in here.